### ISSUE TITLE
Ability to split, merge and re-order cells

### ISSUE THREAD

**Issue by raproth** [polynote/polynote#886]
I miss advanced  cell editing features like split, merge and also re-order cells. 

For example if I copy-paste some code from other sources, I'd like to be able the split the cell into smaller chunks. Is this even possible with the queuing-model in polynote?

**Comment by jeremyrsmith** [comment]
Certainly it would be possible to split cells – it's just inserting a cell below and moving some code to it, right? We just don't have a specialized UI around that.

The tricky part about it is that ideally you'd want to do this without changing the runtime state. What to do with the runtime state (if cells have already been run) is why we've been punting on moving cells – ideally you'd want to disallow or warn if the user tries to move the cell above one of its input dependencies.

**Comment by raproth** [comment]
Yes, changing the runtime state is an issue. We are often struggeling with notebooks which seem to "work", but after a restart they don't work anymore.

We could either show a warning, or only allow these operation if the kernel is stopped
